tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1500,14 +1500,14 @@ def test_log_softmax_invalid(device, a_shape, axis, dtype):
     # Special shapes
     chainer.testing.product({
         'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
-        'dtype': _expected_float_dtypes_math_functions,
+        'in_dtypes,out_dtype': _in_out_float_dtypes_math_functions,
```

Comment:
Should be `_in_out_dtypes_math_functions` based on other tests, however there is a type mismatch between `numpy` and `chainerx` So have kept it as `_in_out_float_dtypes_math_functions` as was the case before this diff.

Reply:
- Thanks for explaining the remaining issue.
- The PR seems a dup of #6679.

Reply:
Oh my bad. Did not notice it. Closing this one.